HyperdriveTests/HyperBlueprintTests.swift

```diff
@@ -59,4 +59,27 @@ class HyperBlueprintTests: XCTestCase {
 
     XCTAssertEqual(representor.attributes as NSDictionary, attributes)
   }
+
+  func testConstructingResponseShowsWebLinkingHeaders() {
+    let attributes = ["question": "Favourite Programming Language?"]
+    let URL = NSURL(string: "https://polls.apiblueprint.org/questions/5")!
+    let request = NSURLRequest(URL: URL)
+    let headers = [
+      "Content-Type": "application/json",
+      "Link": "<https://polls.apiblueprint.org/questions/6>; rel=\"next\", <https://polls.apiblueprint.org/questions/4>; rel=\"prev\"; type=\"foo\"",
```

Comment:
Mostly using a separate library I built for this before under the hood -> https://github.com/kylef/WebLinking.swift